bin/citrea/src/rollup/mod.rs

```diff
@@ -37,15 +38,21 @@ pub trait CitreaRollupBlueprint: RollupBlueprint {
         // Maybe whole "prev_root" can be initialized inside runner
         // Getting block here, so prover_service doesn't have to be `Send`
 
-        let ledger_db = self.create_ledger_db(&rollup_config);
+        let (soft_confirmation_tx, _) = broadcast::channel(10);
```

Comment:
The channel size of 10 i believe should be either configurable or a much larger number

Reply:
IMHO it doesn't make sense to allow user to configure this. We can definitely increase this number, but I am not sure why you think it should much larger? This is only the size of the message buffer, meaning that the requirement for buffer size doesn't increase with the number of consumers. So one block comes to this buffer, is cloned and sent to each consumer, and dropped from the buffer.

Reply:
It is not the size of the message buffer. 
https://docs.rs/tokio/latest/tokio/sync/broadcast/fn.channel.html

A channel's capacity is how many messages you can push into the channel before `send` starts blocking.
> The channel is created with a capacity of 32. If messages are sent faster than they are received, the channel will store them. Once the 32 messages are stored in the channel, calling send(...).await will go to sleep until a message has been removed by the receiver.

Reply:
I don't think that is the case. We have only 1 sender, and we will push to this channel newly created blocks 1 by 1. And internally this broadcast channel will distribute that message to each receiver. It wouldn't make sense for it to create the same message for 100 subscriber, and push 100 of the same message. So only send rate matters, consumer doesn't matter (or it matters as much as it prolongs the blocking `tx.send()` call).

I tested it just in case with the following piece of code:
```rs
#[tokio::test(flavor = "multi_thread")]
async fn test_sub() -> anyhow::Result<()> {
    let (tx, _) = broadcast::channel(1);

    let msg_count = 3;

    // Run 10 consumers
    for i in 0..10 {
        let mut rx = tx.subscribe();
        tokio::spawn(async move {
            for _ in 0..msg_count {
                let value = rx.recv().await.unwrap();
                println!("Consumer {} received value {}", i, value);
            }
        });
    }

    // Run the producer
    for i in 0..msg_count {
        tx.send(i).unwrap();
        sleep(Duration::from_millis(10)).await;
    }

    Ok(())
}
```
and the output is:
```
Consumer 4 received value 0
Consumer 1 received value 0
Consumer 6 received value 0
Consumer 5 received value 0
Consumer 0 received value 0
Consumer 3 received value 0
Consumer 2 received value 0
Consumer 7 received value 0
Consumer 8 received value 0
Consumer 9 received value 0
Consumer 4 received value 1
Consumer 6 received value 1
Consumer 0 received value 1
Consumer 1 received value 1
Consumer 8 received value 1
Consumer 9 received value 1
Consumer 3 received value 1
Consumer 7 received value 1
Consumer 5 received value 1
Consumer 2 received value 1
Consumer 4 received value 2
Consumer 0 received value 2
Consumer 8 received value 2
Consumer 9 received value 2
Consumer 3 received value 2
Consumer 7 received value 2
Consumer 2 received value 2
Consumer 5 received value 2
Consumer 6 received value 2
Consumer 1 received value 2
```

I also tested with 100 consumers, all of them still received all messages.

Reply:
I stand corrected. Thank you for putting the effort to show this.